docs/sources/reference/components/otelcol/otelcol.receiver.influxdb.md

```diff
@@ -26,13 +26,13 @@ otelcol.receiver.influxdb "influxdb_metrics" {
 
 `otelcol.receiver.influxdb` supports the following arguments:
 
-| Name                     | Type           | Description                                                     | Default                                                    | Required |
-| ------------------------ | -------------- | --------------------------------------------------------------- | ---------------------------------------------------------- | -------- |
-| `endpoint`               | `string`       | `host:port` to listen for traffic on.                           | `"localhost:8086"`                                         | no       |
-| `max_request_body_size`  | `string`       | Maximum request body size the server will allow.                | `20MiB`                                                    | no       |
-| `include_metadata`       | `boolean`      | Propagate incoming connection metadata to downstream consumers. |                                                            | no       |
-| `compression_algorithms` | `list(string)` | A list of compression algorithms the server can accept.         | `["", "gzip", "zstd", "zlib", "snappy", "deflate", "lz4"]` | no       |
-`auth`              | `capsule(otelcol.Handler)` | Handler from an `otelcol.auth` component to use for authenticating requests.     |               | no
+| Name                     | Type                       | Description                                                                  | Default                                                    | Required |
```

Comment:
Should this be a separate PR?

Reply:
Yeah, I don't know why I changed this file. Must have been by mistake. Will restore it